dhis-2/dhis-test-performance/docker-compose.profile.yml

```diff
@@ -7,11 +7,15 @@ services:
     security_opt:
       - seccomp:unconfined # Allow profiling syscalls
     environment:
-      JAVA_OPTS:
+      # JAVA_TOOL_OPTIONS: Standard JVM environment variable, works with all image types
+      # Note: Must include all JVM options from base docker-compose.yml since environment is replaced, not merged
+      JAVA_TOOL_OPTIONS:
         "-Dlog4j2.configurationFile=/opt/dhis2/log4j2.xml \
+        -Xmx10000m -Xms10000m \
```

Comment:
This heap mem parameters extracted from somewhere else? Or arbitratily chosen now? Just curious.

Reply:
based on the server we have https://github.com/dhis2/dhis2-core/actions/runs/19856507939 and the fact it shares it with postgres. The `dhis.conf`, the `postgresql.conf` and container resource allocation is definitely something we could tune and verify its the best we can do.